fetch/cosmwasm_contract/examples/schema.rs

```diff
@@ -9,16 +9,16 @@ use bridge::state::State;
 */
 
 fn main() {
-/*
-    let mut out_dir = current_dir().unwrap();
-    out_dir.push("schema");
-    create_dir_all(&out_dir).unwrap();
-    remove_schemas(&out_dir).unwrap();
+    /*
```

Comment:
Will add an item on to the backlog to add this back in at some point

Reply:
added back, that wasn't too hard :)